web/src/engine/predictive-text/templates/src/trie-model.ts

```diff
@@ -303,6 +303,18 @@ export default class TrieModel implements LexicalModel {
   }
 
   predict(transform: Transform, context: Context): Distribution<Suggestion> {
+    //#region Helper function
```

Comment:
I think delete the region markers. It's a local function anyway, and code browser will allow collapse of it.

```suggestion
```